src/CommunityCommons/javasource/unittesting/proxies/microflows/Microflows.java

```diff
@@ -1,137 +1,137 @@
-	{
-		Map<java.lang.String, Object> params = new HashMap<>();
-		params.put("Message", _message);
-		Core.microflowCall("UnitTesting.ReportStep").withParams(params).execute(context);
-	}
-	public static void rerunUnittest(IContext context, unittesting.proxies.UnitTest _unitTestRun)
-	{
-		Map<java.lang.String, Object> params = new HashMap<>();
-		params.put("UnitTestRun", _unitTestRun == null ? null : _unitTestRun.getMendixObject());
-		Core.microflowCall("UnitTesting.RerunUnittest").withParams(params).execute(context);
-	}
-	public static void setup(IContext context)
-	{
-		Map<java.lang.String, Object> params = new HashMap<>();
-		Core.microflowCall("UnitTesting.Setup").withParams(params).execute(context);
-	}
-	public static void startUnittestRun(IContext context, unittesting.proxies.TestSuite _testSuite)
-	{
-		Map<java.lang.String, Object> params = new HashMap<>();
-		params.put("TestSuite", _testSuite == null ? null : _testSuite.getMendixObject());
-		Core.microflowCall("UnitTesting.StartUnittestRun").withParams(params).execute(context);
-	}
-	public static boolean startup(IContext context)
-	{
-		Map<java.lang.String, Object> params = new HashMap<>();
-		return (java.lang.Boolean) Core.microflowCall("UnitTesting.Startup").withParams(params).execute(context);
-	}
-	public static void tearDown(IContext context)
-	{
-		Map<java.lang.String, Object> params = new HashMap<>();
-		Core.microflowCall("UnitTesting.TearDown").withParams(params).execute(context);
-	}
-	public static boolean test_FailingUnitTest(IContext context)
-	{
-		Map<java.lang.String, Object> params = new HashMap<>();
-		return (java.lang.Boolean) Core.microflowCall("UnitTesting.Test_FailingUnitTest").withParams(params).execute(context);
-	}
-	public static boolean test_IsAssertionErrorCheck(IContext context)
-	{
-		Map<java.lang.String, Object> params = new HashMap<>();
-		return (java.lang.Boolean) Core.microflowCall("UnitTesting.Test_IsAssertionErrorCheck").withParams(params).execute(context);
-	}
-	public static boolean test_ValidUnitTest(IContext context)
-	{
-		Map<java.lang.String, Object> params = new HashMap<>();
-		return (java.lang.Boolean) Core.microflowCall("UnitTesting.Test_ValidUnitTest").withParams(params).execute(context);
-	}
-	public static void test_WithoutReturn(IContext context)
-	{
-		Map<java.lang.String, Object> params = new HashMap<>();
-		Core.microflowCall("UnitTesting.Test_WithoutReturn").withParams(params).execute(context);
-	}
-	public static void unitTestOverview(IContext context)
-	{
-		Map<java.lang.String, Object> params = new HashMap<>();
-		Core.microflowCall("UnitTesting.UnitTestOverview").withParams(params).execute(context);
-	}
-	public static boolean uT_ValidUnitTest(IContext context)
-	{
-		Map<java.lang.String, Object> params = new HashMap<>();
-		return (java.lang.Boolean) Core.microflowCall("UnitTesting.UT_ValidUnitTest").withParams(params).execute(context);
-	}
+// This file was generated by Mendix Studio Pro.

```

Comment:
_Nit:_ The file now uses **CRLF** instead of **LF**. I don't know if this is an intentional change, or whether it is something we should even take a look at (considering that many other files use CRLF as well).

I'll leave the decision whether to ignore this up to you.

Reply:
I changed that because studio pro generates them this way, and I thought changes every time run press 'run locally' are quite annoying. I'm tempted to change all of them to CRLF in the future.

Reply:
Unless I should just configure git correctly of course ;)

Reply:
In the last commit I've added all files as studio pro generates them, that at least makes it consistent.

Reply:
Agreed, this looks a lot better, thanks!